src/ram/analysis/Complexity.cpp

```diff
@@ -62,7 +62,7 @@ int ComplexityAnalysis::getComplexity(const Node* node) const {
         }
 
         int visit_(type_identity<UserDefinedOperator>, const UserDefinedOperator&) override {
-            return 10;
```

Comment:
As per the comment, make it the maximum value of Int.

Reply:
Done.